library/tasks/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -0,0 +1,88 @@
+
+task CalculateGeneMetrics {
+  File bam_input
+  Int estimated_required_disk = ceil(size(bam_input, "G") * 2)
+
+  command {
+    CalculateGeneMetrics -i "${bam_input}" -o gene-metrics.csv
+    gzip gene-metrics.csv
+  }
+
+  runtime {
+    docker: "quay.io/humancellatlas/secondary-analysis-sctools:ajc-merge-metrics"
```

Comment:
I would change these docker images to the latest version of `secondary-analysis-sctools` now that your `ajc-merge-metrics` PR has been merged